manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2112.20
+PackageLocale: en-US
+Publisher: gdiObjects
+PublisherUrl: https://www.gdiobjects.com/index.php
+PublisherSupportUrl: https://www.gdiobjects.com/helpdesk/
+PrivacyUrl: https://www.gdiobjects.com/privacy.php
+Author: gdiObjects
+PackageName: xBar
+PackageUrl: https://www.gdiobjects.com/Products/xBar/Index.php
+License: Copyright © 2008-2021 gdiObjects
+ShortDescription: A Dockable Infobar and App Launcher for Windows
+Description: The Fast & Easy Way to Find and Launch Software Apps, Open Documents, or Access Folders, and Drives in Windows. Quickly Organize Shortcuts into User-defined Categories. View System Details and Monitor your PC’s Performance with CPU, Memory, Temperature, and Battery Sensors.
+Moniker: xBar
+Tags:
+- sidebar
+- appbar
+- windows 10
+- windows 11

```

Comment:
```suggestion
- windows-10
- windows-11
```

Reply:
I would argue that these tags should be removed entirely, as they are what I would consider to be "low-value" tags, in that they don't provide any meaningful information about the application itself or it's uses. They simply denote which versions of windows the application can be installed on, which could easily be covered by adding the MinimumOSVersion